HISTORY.rst

```diff
@@ -6,6 +6,7 @@ History
 v0.32 (unreleased)
 ..................
 * fix json generation for EnumError, #697 by @dmontagu
+* fix ``self`` inspection for smart debuggers, #701 by @peter-boers, @pfrederiks
```

Comment:
Afraid I'm just in the process of releasing v0.31.1 with the `EnumError` fix, you'll need to rebase and fix this.

Reply:
done